docs/csharp/programming-guide/concepts/linq/codesnippet/CSharp/data-transformations-with-linq_3.cs

```diff
@@ -14,8 +14,7 @@ static void Main()
             // Create the query.
             var studentsToXML = new XElement("Root",
                 from student in students
-                let x = String.Format("{0},{1},{2},{3}", student.Scores[0],
-                        student.Scores[1], student.Scores[2], student.Scores[3])
+                let x = $"{student.Scores[0]},{student.Scores[1]},{student.Scores[2]},{student.Scores[3]}"
```

Comment:
This code really assumes that there are exactly four scores, the possible alternative is [string.Join](https://docs.microsoft.com/en-us/dotnet/api/system.string.join?view=netframework-4.7.2#System_String_Join__1_System_String_System_Collections_Generic_IEnumerable___0__)
```csharp
string.Join(",", student.Scores)
```

Reply:
Corrected in 4ca2fdb

Reply:
> Corrected in [4ca2fdb](https://github.com/dotnet/docs/commit/4ca2fdb6a9a00a89b1f0f3997b4ce95a9f90d989)

Thanks!